tasking-mgr-projects/package.json

```diff
@@ -6,13 +6,17 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/AmericanRedCross/osm-data-parse.git"
+  },
   "author": "danbjoseph",
-  "license": "",
+  "license": "ISC",
```

Comment:
Thanks for updating the package.json. However, all of the work that American Red Cross does is BSD-3-Clause.

Reply:
Ok!  Will make the change.  Thanks!